### PR TITLE
#164858926 Create a database model for office structure

### PR DESCRIPTION
**Pull request by kwanj-k**
### What does this PR do?
Adds office_structures table

### Description of the task to be completed?
Create a new office structure model

### How should this be manually tested?
1. Clone the repo: `git clone https://github.com/andela/mrm_api.git`
2. Cd into the cloned repo 
3. Git pull branch ft-office-structure-164858926
4. Run migrations
5. New table - office_structures should be created in the database

### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#164858926](https://www.pivotaltracker.com/story/show/164858926)
### Checklist:
- [x]  My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] My changes generate no new warnings
- [x] Existing unit tests pass locally with my changes
- [x] Implementation works according to expectations

### CODE REVIEW

File: tests/base.py [hunk 1]
```diff
@@ -155,6 +156,16 @@ def setUp(self):
             )
             response_2.save()
             response_2.missing_resources.append(resource)
+            structure = Structure(
+                web_id='gsgsggg',
```
Comment (nakatuddesuzan) [inline tests/base.py]: Hello @kwanj-k , I suggest that we use a more descriptive `web_id` here. You can get a sample of a web id from the front-end and we use that.

File: api/structure/models.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+class Structure(Base, Utility):
+    __tablename__ = "office_structures"
+
+    id = Column(Integer, primary_key=True)
+    # web_id is the id from the client side
+    web_id = Column(String, nullable=False)
+    level = Column(Integer, nullable=False)
+    name = Column(String(50), nullable=False)
+    parent_id = Column(Integer, nullable=False)
+    tag = Column(String(50), nullable=False)
```
Comment (nakatuddesuzan) [inline api/structure/models.py]: About this tag column. Kindly confirm that it is a `nullable=False`.

Comment (kwanj-k) [reply api/structure/models.py]: The front-end dev indicated that all the fields are required @nakatuddesuzan

File: tests/base.py (hunk 1 above)
Comment (kwanj-k) [reply tests/base.py]: @nakatuddesuzan the id comes with the format ``` id: "b05fc5f2-b4aa-4f48-a8fb-30bdcc3fc968"  ``` 
I decided to call it web_id as we have sqlachemy id

File: api/structure/models.py (hunk 2 above)
Comment (nakatuddesuzan) [reply api/structure/models.py]: Noted

**Pull request merged**